README.md

```diff
@@ -210,6 +210,24 @@ The full process is described graphically in the [workflow diagram](./docs/workf
 
   > **NOTE** ⏳ This command can take few minutes or immediately depending on your internet connection or model is cached. If you have issues connecting to Hugging Face, refer to the [Hugging Face discussion forum](https://discuss.huggingface.co/) for more details.
 
+  #### Downloading a specific model from a Hugging Face repository
+
+- Specify repository, model, and if necessary a hugging face token
+
+  ```shell
+  export HF_TOKEN=<YOUR HUGGINGFACE TOKEN GOES HERE>
```

Comment:
If it's used once I won't recommend `export` but would simplify prefixing the `ilab` command with it.